simple-umbraco-webapp/azuredeploy.json

```diff
@@ -0,0 +1,456 @@
+            "type": "string",
+            "metadata": {
+                "description": "Non-admin Database User. Must be Unique"
+            }
+        },
+        "NonAdminDatabasePassword": {
+            "type": "securestring",
+            "metadata": {
+                "description": "Non-admin Database User password"
+            }
+        },
+        "storageAccountType": {
+            "type": "string",
+            "allowedValues": [
+                "Standard_LRS",
+                "Standard_GRS",
+                "Standard_RAGRS",
+                "Standard_ZRS"
+            ],
+            "defaultValue": "Standard_LRS",
+            "metadata": {
+                "description": "Storage Account Type : Standard-LRS, Standard-GRS,Standard-RAGRS,Standard-ZRS"
+            }
+        }
+    },
+    "variables": {
+        "appInsightsRegion": "Central US",
+        "storageAccountName": "[concat(uniquestring(resourceGroup().id), 'standardsa')]",
+        "umbracoAdminWebAppName": "[concat(parameters('appName'), 'adminapp')]",
+        "appServicePlanName":"[concat(parameters('appName'), 'serviceplan')]"
+    },
+    "resources": [       
+        {
+            "name": "[parameters('dbServerName')]",
+            "type": "Microsoft.Sql/servers",
+            "location": "[resourceGroup().location]",
+            "apiVersion": "2014-04-01-preview",
+            "properties": {
+                "administratorLogin": "[parameters('dbAdministratorLogin')]",
+                "administratorLoginPassword": "[parameters('dbAdministratorLoginPassword')]",
+                "version": "12.0"
+            },
+            "resources": [
+                {
+                    "apiVersion": "2014-04-01-preview",
+                    "name": "[parameters('dbName')]",
+                    "type": "databases",
+                    "location": "[resourceGroup().location]",
+                    "dependsOn": [
+                        "[parameters('dbServerName')]"
+                    ],
+                    "properties": {
+                        "edition": "[parameters('dbEdition')]",
+                        "collation": "SQL_Latin1_General_CP1_CI_AS",
+                        "requestedServiceObjectiveId": "F1173C43-91BD-4AAA-973C-54E79E15235B"
+                    }
+                },
+                {
+                    "apiVersion": "2014-04-01-preview",
+                    "name": "AllowAllWindowsAzureIps",
+                    "type": "firewallrules",
+                    "location": "[resourceGroup().location]",
+                    "dependsOn": [
+                        "[concat('Microsoft.Sql/servers/', parameters('dbServerName'))]"
+                    ],
+                    "properties": {
+                        "endIpAddress": "0.0.0.0",
+                        "startIpAddress": "0.0.0.0"
+                    }
+                }
+            ]
+        },
+        {
+            "apiVersion": "2015-05-01-preview",
```

Comment:
Use 2016-01-01 version for storage. There is a breaking change you need to be aware of. See API definition https://msdn.microsoft.com/en-us/library/azure/mt163564.aspx